Kitodo-DataEditor/src/test/resources/testAnExtensiveRulesetCanBeLoaded.xml

```diff
@@ -13,7 +13,7 @@
 -->
 <ruleset xmlns="http://names.kitodo.org/ruleset/v2"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://names.kitodo.org/ruleset/v2 ../../main/resources/ruleset.xsd">
+         xsi:schemaLocation="http://names.kitodo.org/ruleset/v2 ../../../../Kitodo/rulesets/ruleset.xsd">
```

Comment:
I thought, if it is in the same folder, there is no path required?

Reply:
We have rule sets in **two** different folders: One to use and one for the tests. In one of both folders you logically need the path to the other folder. Alternative would be, of course, to copy the file into each folder. Personally, I am against it, because that brings with it all the problems of duplicate code. Say, if someone changes one of the two files you do not know in the end which is right now.

Reply:
Tests should be independent from real directories. So in this case copy would be suitable solution.